.ci/Jenkinsfile

```diff
@@ -267,9 +267,7 @@ def doSlackSend(String color) {
     testsSuites = "All suites"
   }
 
-  def message = "*[${testsSuites}]* Build `${buildStatus}`: ${env.JOB_NAME} ${env.BUILD_NUMBER} (<${env.RUN_DISPLAY_URL}|Open>)."
-
-  slackSend(channel: "#${env.SLACK_CHANNEL}", color: "${color}", message: "${message}",tokenCredentialId: 'jenkins-slack-integration-token')
+  notifyBuildResult(prComment: true, slackHeader: "*Suite*: ${testSuites}", slackChannel: "${env.SLACK_CHANNEL}", slackComment: true, slackNotify: notify)
```

Comment:
Not sure if this `prComment` and the one in https://github.com/elastic/e2e-testing/pull/383/files#diff-ef8b2f5e4d21d81c0eba4754dea3a3dfc2c67d26284f195165b650cf4aff7beaR234 will collide